common/interest_group_traits/00_trade_unions_traits.txt

```diff
@@ -3,7 +3,7 @@
 	min_approval = loyal
 	modifier = {
 		building_economy_of_scale_level_cap_add = 5
-		state_dependent_wage_mult = 0.1
+		state_political_strength_from_welfare_mult = 0.05
```

Comment:
This was already an edit of the original version which only gives a working adult ratio. However, the French unique trait paradox developed indicated that something like political strength from welfare was a desirable trait they want to allow Trade Unions to have, so i felt it would be more appropriate. If you feel its more appropriate though we can just change it back.

Reply:
Nah lets keep the change then.